index.js

```diff
@@ -216,25 +216,25 @@ async function main (argv) {
     process.exit(1)
   }
 
-  process.stdout.write('Ok.')
+  process.stdout.write('Ok.\n')
 
   //
   // Initialize the current directory as a socket app.
   //
   try {
-    process.stdout.write('\nCreating socket files...')
+    process.stdout.write('Creating socket files...')
```

Comment:
Just checking, this one doesn't need a trailing `\n`?

Reply:
It doesn't need.
```
Creating socket files...Ok.
```